pkg/proto/operator/v1/operator.pb.go

```diff
@@ -33,6 +33,63 @@ const (
 	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
 )
 
+// ResourceEventType is the type of event to a resource.
+type ResourceEventType int32
+
+const (
+	// UNKNOWN indicates that the event type is unknown.
+	ResourceEventType_UNKNOWN ResourceEventType = 0
```

Comment:
Since the type is ResourceEventType, prefix is not necessary.

Reply:
This is generated code which is coming from the proto defined [here](https://github.com/dapr/dapr/pull/6845/files#diff-4e6e6e3126ef3d277ca245bc39a810828410d85a49c06024a9c1f0fd42a1b359R44).